lib/network_manager.go

```diff
@@ -177,6 +177,19 @@ func (nm *NetworkManager) Stop() {
 // ## NetworkManager Routines
 // ###########################
 
+func (nm *NetworkManager) printActiveValidators() {
```

Comment:
All of this logging was not actually that useful because it just prints the full validator set over and over again (doesn't distinguish between who you're connected to vs just "aware" of). I'd delete everything around printActiveValidators().